app/views/unattended/provisioning_templates/snippet/preseed_networking_setup.erb

```diff
@@ -45,10 +45,11 @@ EOF
 <% interface_subnet6 = interface.subnet6 -%>
 <% interface_dhcp6 = interface_subnet6.nil? ? true : interface_subnet6.dhcp_boot_mode? -%>
 <% next if !interface.managed? || (interface_subnet.nil? && interface_subnet6.nil?) || interface.primary -%>
-real=`ip -o link | awk '/<%= interface.mac -%>/ {print $2;}' | sed s/:$//`
 <% virtual = interface.virtual? -%>
```

Comment:
@ekohl I think I should have removed `<% virtual = interface.virtual? -%>` as well, not sure why it is still there. Shall I create a fixup?

Reply:
Oh yes, it could be removed now. Good catch.